### ISSUE TITLE
Mods with an error are at the top

### ISSUE THREAD

**Issue by Quickness19** [studio-minus/ppg-bugs-and-requests#872]
**Feature description**
Handy stuff.

**Feature purpose**
Mods with an error are at the top. It will help when creating mods. You don't have to type in the name or scroll down. Quick and convenient.

**Additional context**
none.

**Comment by DindinYT37** [comment]
I agree and I think there should generally be mod filters or mod sorting (like Alphabetically, Disabled, Enabled, Error, Local, etc.)
